### DISCUSSION TITLE
{建议}在游戏内改变角色属性，比如鹅变鸭，鹅变警长

### DISCUSSION THREAD

**Discussion by heixiu03** [FrozenFish259/Goose_Goose_Duck_Hack#91] [Ideas 分享新功能的灵感和想法]

**Comment by heixiu03** [comment]
幽灵变活人，活人变幽灵，可能局限于本地

**Comment by AK47donk** [comment]
这个想法很疯狂，可以试试哈哈哈

**Comment by pangjiea** [comment]
hh，我想知道房主对于游戏数据的权限和普通成员一样吗

**Comment by heixiu03** [comment]
> 
我在ce里面试过没找到角色属性

**Comment by FrozenFish259** [comment]
> hh，我想知道房主对于游戏数据的权限和普通成员一样吗

不是P2P游戏，没有不对等关系

**Comment by FrozenFish259** [comment]
> hh，我想知道房主对于游戏数据的权限和普通成员一样吗

要是房主是服务器主机就好玩了哈哈

**Comment by heixiu03** [comment]
> > hh，我想知道房主对于游戏数据的权限和普通成员一样吗
> 
> 要是房主是服务器主机就好玩了哈哈

房主应该不是服务器主机，因为房主退了，游戏还是会继续，或许是开始游戏之后，所有人都有房间数据核对
